sim/hunter/aimed_shot.go

```diff
@@ -56,7 +56,7 @@ func (hunter *Hunter) getAimedShotConfig(rank int, timer *core.Timer) core.Spell
 			},
 		},
 		ExtraCastCondition: func(sim *core.Simulation, target *core.Unit) bool {
-			return hunter.DistanceFromTarget >= 8
+			return hunter.DistanceFromTarget >= 12
```

Comment:
Maybe we could pull this out into a constant value in `sim/core/constants`? `core.MinRangedAttackDistance`

Reply:
Added constant for both MaxMeleeAttackDistance and MinRangedAttackDistance